internal/errors/errors.go

```diff
@@ -158,7 +158,7 @@ const (
 	// LevelDBFailedRetriveOriginalKey problem retrieving entry - original key
 	LevelDBFailedRetriveOriginalKey = "Failed to retrieve the entry for the original key: %s. %s"
 	// LevelDBFailedRetriveGeneratedID problem retrieving entry - generated ID
-	LevelDBFailedRetriveGeneratedID = "Failed to retrieve the entry for the generated ID: %s. %s"
+	LevelDBFailedRetriveGeneratedID = "failed to retrieve the entry for the generated ID: %s. %s"
```

Comment:
Just curious why this one was lowercased but not the others?

Reply:
That's because I am getting this:
```
internal/rest/receipt/leveldb.go:227:15: ST1005: error strings should not be capitalized (stylecheck)
                return nil, fmt.Errorf(errors.LevelDBFailedRetriveGeneratedID, requestID, err)
```